scripts/testing/check_top.sh

```diff
@@ -2,12 +2,26 @@
 
 # standard bash error handling
 set -o nounset  # treat unset variables as an error and exit immediately.
-set -o errexit  # exit immediately when a command fails.
 set -E          # needs to be set if we want the ERR trap
 set -o pipefail # prevents errors in a pipeline from being masked
 
-echo -e "\n--- BTP Manager checking kubectl top"
-kubectl top pod -l app.kubernetes.io/component=btp-manager.kyma-project.io -n kyma-system --containers
+TIMEOUT=30
+NEXT_TRY_WAIT=5
 
-echo -e "\n--- BTP Operator checking kubectl top"
-kubectl top pod -l app.kubernetes.io/name=sap-btp-operator -n kyma-system --containers
\ No newline at end of file
+echo -e "\n--- BTP Manager checking kubectl top" 
```

Comment:
Code duplication - you execute almost the same code twice.
The only difference is pod name.